app/models/import_export/import_xls.rb

```diff
@@ -70,5 +52,18 @@ def message_erreur_validation(exception, row)
     def self.fichier_xls?(file)
       file.content_type == XLS_MIME_TYPE && File.extname(file.original_filename) == '.xls'
     end
+
+    def self.telecharge_fichier(url, nom_technique)
+      fichier = Down.download(url)
+      extention = File.extname(url)
+      nom_fichier = "#{nom_technique}#{extention}"
+      content_type = Marcel::MimeType.for(fichier.path, name: nom_fichier)
```

Comment:
Je me demande si ça ne devrait pas être `Marcel::MimeType.for(fichier, name: nom_fichier)` (sans `.path`)

J'ai laissé comme c'était, mais si je comprend ce que fait Marcel::MimeType, il a l'air de pouvoir retrouver le type de contenu à partir du contenu lui-même.

Reply:
Ou alors on pourrait simplifier avec : `Marcel::MimeType.for extension: ".pdf"`

Reply:
J'ai choisi de me basé uniquement sur l'extension.